lms/templates/navigation.html

```diff
@@ -124,15 +124,21 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
         % endif
       </%block>
       % if not settings.FEATURES['DISABLE_LOGIN_BUTTON']:
+          % if settings.FEATURES.get('ENABLE_COURSE_DISCOVERY'):
+          <li class="nav-global-05">
+            <a class="cta cta-discovery" href="/courses">${_("Find Courses")}</a>
+          </li>
+          %endif
           % if course and settings.FEATURES.get('RESTRICT_ENROLL_BY_REG_METHOD') and course.enrollment_domain:
```

Comment:
As above, it would be better to move this logic into the view code and set a context parameter like 'show_registration_link" to True/False